axiom/tokens.go

```diff
@@ -0,0 +1,239 @@
+package axiom
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"time"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+//go:generate go run golang.org/x/tools/cmd/stringer -type=Action -linecomment -output=action_string.go
```

Comment:
One last favour: Please rename this file to `tokens_string.go`. I get your intention - to name it after the only type it contains - but naming it `tokens` groups it nicely with all its related files (and is resistant to new types being added in the future and the filename suddenly not making sense anymore 😄)